CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ endif()
 option (REPO_BUILD_COVERAGE "Building with coverage report" OFF)
 option (ODA_SUPPORT "If ODA library is present" OFF)
 option (SYNCHRO_SUPPORT "If Synchro library is present" OFF)
-option (VALID_UNTIL "Set timestamp for how long (sec since epoch) this library should be valid for (-1 for unlimited) " -1)
+option (CRYPTOLENS_LICENSE "Build with cryptolens license check" $ENV{ENABLE_LICENSE_CHECK})
```

Comment:
don't do that, just let it be OFF like everything else, it doesn't need to be an env var.

Reply:
Let's only reference CRYPTO if it's specifically about their library. LICENSE_CHECK should suffice.